repositories/permissions_requests.go

```diff
@@ -0,0 +1,121 @@
+package repositories
+
+import "github.com/topfreegames/Will.IAM/models"
+
+// PermissionsRequests repository
+type PermissionsRequests interface {
+	Clone() PermissionsRequests
+	Create(*models.PermissionRequest) error
+	Deny(string, string) error
```

Comment:
Name the parameters? (string, string) is too ambiguous. Same for others.

Most methods here could benefit from a comment explaining what they do, too.